Sources/FunctionalTableData/TableView/FunctionalTableData+UITableViewDelegate.swift

```diff
@@ -152,6 +152,14 @@ extension FunctionalTableData {
 			}
 			
 			if let cellConfig = data.sections[indexPath] {
+				switch cellConfig.actions.selectionState {
+				case .selected:
+					tableView.selectRow(at: indexPath, animated: false, scrollPosition: .none)
+				case .deselected:
+					tableView.deselectRow(at: indexPath, animated: false)
```

Comment:
Any reason we don't want the selection / deselection to be animated?

Reply:
Yes! Since we're in `willDisplay cell`, the times when this is called and we actually want to select a cell will likely *not* be the first time this cell is presenting (ie reload, scrolling, paginating new cells into the table). I figure if someone wanted to select it the first time the cell shows up and animate it then can just have it show up as deselected and then select the cell by calling `tableView.selectRow(at indexPath, animated: true...)`